scripts/masterCron.ts

```diff
@@ -204,6 +172,8 @@ async function settleAuction(auctionCounters: Array<number>) {
     network,
     "burnRemainingOTokens"
   );
+
+  auctionIDs = [];
```

Comment:
Do we use this anywhere?

Reply:
ye this is a global. rolltonextoption populates with id's. in settleAuction they are used to settle correct auctions. then they are cleared. repeat

Reply:
https://github.com/ribbon-finance/ribbon-v2/blob/b66275966278a0d7ac447a2afbc9c102ae12d1f6/scripts/masterCron.ts#L115

Reply:
What if it fails for whatever reason? We will be losing the auction IDs needed to settle. We should persist it somewhere imo

Reply:
Maybe use LevelDB or something? https://github.com/google/leveldb

Reply:
Better yet - we can just expose this info on the contract itself. Then the cron script can lookup the active auctions.